code/modules/mob/living/carbon/xenomorph/castes/widow/abilities_widow.dm

```diff
@@ -234,7 +234,8 @@
 		INVOKE_ASYNC(src, PROC_REF(xeno_burrow_doafter))
 		return
 	UnregisterSignal(X, COMSIG_XENOMORPH_TAKING_DAMAGE)
-	X.fire_resist_modifier += BURROW_FIRE_RESIST_MODIFIER
+	X.soft_armor = X.soft_armor.modifyRating(fire = 100)
+	X.hard_armor = X.hard_armor.modifyRating(fire = 100)
```

Comment:
Should this add trait_non_flammable, or will the armor handle that enough?

Reply:
yeah good catch.